packages/app/src/lib/workers/materializer.ts

```diff
@@ -308,6 +322,16 @@ const materializers: {
           ${data.content.mimeType},
           ${data.content.content}
         )`,
+      sql`
+        insert into comp_last_read (entity, timestamp, unread_count)
+        values (${id(event.parent)}, 1, 1)
+        on conflict(entity) do update set
+          unread_count = case
+            when ${id(event.ulid)} > timestamp_to_ulid(comp_last_read.timestamp)
```

Comment:
Can we we just `decodeTime()` from the `event.ulid` before comparing it with `comp_latest_read.timestamp` here?

Converting a timestamp to a ULID just for comparison is a little confusing to me, and using the UDF will also be a little slower than just pre-parsing the event ULID before sending it to the SQL query.